lib/ui/views/forum/forum-post-feed/forum_post_feed_view.dart

```diff
@@ -51,97 +52,73 @@ class ForumPostFeedView extends StatelessWidget {
               SchedulerBinding.instance!.addPostFrameCallback(
                   (timeStamp) => model.handlePostLazyLoading(index));
             },
-            child: postViewTemplate(post![index], index, model)));
+            child: InkWell(
+              onTap: () {
+                model.navigateToPost(
+                  post[index].postSlug,
+                  post[index].postId,
+                );
+              },
+              child: Container(
+                child: Padding(
+                  padding: const EdgeInsets.only(bottom: 24.0),
```

Comment:
Is there a reason there's padding here? If required then wouldn't it be better if the contents are in the center and not located at the top.

<img width="361" alt="Screen Shot 2021-11-03 at 14 26 48" src="https://user-images.githubusercontent.com/36357875/140044368-0cd78b16-08dd-42b0-84ca-94ac4bec8dbb.png">